lib/config/presets/local/index.ts

```diff
@@ -9,12 +9,12 @@ import * as gitlab from '../gitlab';
 import type { Preset, PresetConfig } from '../types';
 
 const resolvers = {
-  [PlatformId.Azure]: azure,
-  [PlatformId.Bitbucket]: bitbucket,
-  [PlatformId.BitbucketServer]: bitbucketServer,
-  [PlatformId.Gitea]: gitea,
-  [PlatformId.Github]: github,
-  [PlatformId.Gitlab]: gitlab,
+  ['azure']: azure,
```

Comment:
```suggestion
  azure: azure,
```
simplify 😉

Reply:
TS Error: The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type

Reply:
should probably only be `azure`. you need to change all those assignments